playlists/utils.py

```diff
@@ -0,0 +1,24 @@
+import random
+
+from playlists.models import Playlist
+
+
+class PlaylistUtils:
+    def create_random(self, user):
+        result_playlist = Playlist(
+            name="True Random", description="Created with Derek's Spotify Tool"
+        )
+        tracks = user.library.tracks.order_by('?')
+        result_playlist.tracks = tracks
+        return result_playlist
+
+    def create_by_artists(self, *, user, artists, name):
+        description = "Created by Derek's App"
+        result_playlist = Playlist(name=name, description=description)
+        lib_tracks = list(user.library.tracks.filter(artists__in=artists))
+        tracks = []
```

Comment:
Is this needed at all? This looks like legacy from the old way. `tracks` and `lib_tracks` appear to be the same now. So re-building this looks to be overkill.